views/templates/modals/modal-update.html.twig

```diff
@@ -1,11 +1,21 @@
 {% extends "@ModuleAutoUpgrade/components/modal.html.twig" %}
 
+{% set title = 'Start update?'|trans({}) %}
+
 {% block modal_content %}
+  <form 
+    class="update-backup-page__field-list"
+    action=""
+    data-route-to-submit="{{ form_route_to_confirm }}"
+    id="form-confirm-update"
+    name="form-confirm-update"
+  >
```

Comment:
@tblivet By wrapping the form contents in a `form`, I've lost some space between the paragraph and the next input.

![image](https://github.com/user-attachments/assets/e850e8e0-8bda-4b74-b38f-f20c9a954046)

Does this ring a bell?